### PR TITLE
First draft documentation

### PR DESCRIPTION
**Pull request by wsphillips**

### CODE REVIEW

File: docs/src/core/gates.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Gates
+## Overview
+Gates are the most basic building block for describing dynamics in Conductor.jl. A gate
+integrates zero or more inputs into a single unitless weighting value. If desired, we can
+explicitly define the symbolic expression that models the output of a gate:
+
+```julia
```
Comment (ChrisRackauckas) [inline docs/src/core/gates.md:7]:
```suggestion
```@example gates
```

File: docs/src/core/gates.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    # sigmoid(t) ~ 1 / (1 + exp(-Vₘ(t)))
+```
+
+Alternatively, we can take advantage of customized constructors, which let us describe gate
+dynamics in a more canonical, domain-friendly form. For instance, in the classic
+Hodgkin-Huxley formalism, gates are analogous to the "gating particles" used to describe the
+kinetics of voltage-gated ion channels. In literature, ionic channel kinetics are often
+described in terms of forward (``\alpha``) and reverse (``\beta``) reaction rates:
+
+```julia
```
Comment (ChrisRackauckas) [inline docs/src/core/gates.md:27]: etc

File: docs/make.jl [hunk 3]
```diff
@@ -3,7 +3,12 @@ using Documenter, Conductor
 makedocs(sitename="Conductor.jl",
          doctest = false,
          modules = [Conductor],
```
Comment (ChrisRackauckas) [inline docs/make.jl:5]: https://github.com/SciML/DiffEqFlux.jl/pull/721/files#diff-4aae2d1c783cade58bd2cb13748da956e568b7f2aed5fafd9e2a46fb97daf613R14-R21 could be helpful

Comment (wsphillips) [reply docs/make.jl:5]: Doc tests are enabled and strict errors are enabled for all checks. All examples are setup to use `jldoctest` or, in some cases, `@example` paired with hidden `@assert` lines.

**Pull request merged**